src/interface/thirdparty-pisp-api.yaml

```diff
@@ -1666,6 +1780,74 @@ paths:
           $ref: '#/paths/~1consents/post/responses/501'
         '503':
           $ref: '#/paths/~1consents/post/responses/503'
+    patch:
+      description: |
+        The HTTP request `PATCH /consents/{ID}` is used to notify the PISP and DFSP of revoked consent.
+
+        Switch -> PISP
+      summary: NotifyConsent
+      tags:
+        - consent
+      operationId: NotifyConsent
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: ConsentsIDPatchResponse
+              description: |
+                PATCH /consents/{ID} request object.
+
+                Sent to both the PISP and DFSP when a consent is revoked.
+              type: object
+              properties:
+                status:
+                  title: ConsentStatusType
+                  type: string
+                  enum:
+                    - ACTIVE
```

Comment:
Is there any reason for this to be `ACTIVE` still? I can't see any in the sequences: https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#2-unlinking

Reply:
No. There shouldn't be...